src/components/portfolio/DetailsVolatilityGraph.tsx

```diff
@@ -82,8 +82,8 @@ const DetailsVolatilityGraph: React.FC<DetailsVolatilityGraphProps> = ({
               tooltipText={symbol}
               color={
                 volatility > 1
-                  ? theme.palette.error.main
-                  : theme.palette.success.main
+                  ? theme.palette.lightBlue.main
+                  : theme.palette.secondary.light
```

Comment:
I think the colors should be swapped unless blue = bad and pink = good?

Reply:
I forgot we said that we'd keep red and green here, sorry. I changed it back.